sdk/maps/azure-maps-search/azure/maps/search/_search_client.py

```diff
@@ -0,0 +1,911 @@
+if TYPE_CHECKING:
+    from azure.core.credentials import TokenCredential
+
+
+# By default, use the latest supported API version
+class MapsSearchClient(MapsSearchClientBase):
+    """Azure Maps Search REST APIs.
+    :param credential: Credential needed for the client to connect to Azure.
+    :type credential: ~azure.core.credentials.TokenCredential
+    :keyword api_version:
+            The API version of the service to use for requests. It defaults to the latest service version.
+            Setting to an older version may result in reduced feature compatibility.
+    :paramtype api_version: str or ~azure.ai.translation.document.MapsSearchApiVersion
+    """
+
+    def __init__(
+        self,
+        credential, # type: Union[AzureKeyCredential, TokenCredential]
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> None
+
+        super().__init__(
+            credential=credential, **kwargs
+        )
+
+    @distributed_trace
+    def get_geometries(
+        self,
+        geometry_ids,  # type: List[str]
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> List[Polygon]
+        """**Get Polygon**
+        `Reference Document <https://docs.microsoft.com/en-us/rest/api/maps/search/get-search-polygon>`_.
+
+        :param geometry_ids: Comma separated list of geometry UUIDs, previously retrieved from an
+         Online Search request.
+        :type geometry_ids: list[str]
+        :return: List[Polygon], or the result of cls(response)
+        """
+        polygon_result = self._search_client.list_polygons(
+            geometry_ids,
+            **kwargs
+        )
+        result = [] if not polygon_result else polygon_result.polygons
+        return result
+
+    @overload
+    def fuzzy_search(
+        self,
+        query,  # type: str
+        *,
+        coordinates,  # type: Union[str, LatLon]
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> "SearchAddressResult"
+        """
+        :param query: The applicable query string (e.g., "seattle", "pizza"). Can *also* be specified
+         as a comma separated string composed by latitude followed by longitude (e.g., "47.641268,
+         -122.125679"). Must be properly URL encoded.
+        :type query: str
+        :param coordinates: coordinates
+        :type coordinates: ~azure.maps.search._models.LatLon
+        """
+
+    @overload
+    def fuzzy_search(
+        self,
+        query,  # type: str
+        *,
+        country_filter,  # type list[str]
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> "SearchAddressResult"
+        """
+        :param query: The applicable query string (e.g., "seattle", "pizza"). Can *also* be specified
+         as a comma separated string composed by latitude followed by longitude (e.g., "47.641268,
+         -122.125679"). Must be properly URL encoded.
+        :type query: str
+         :param country_filter: Comma separated string of country codes, e.g. FR,ES. This will limit the
```

Comment:
The line is incorrectly indented. Also not a comma separated string.
Lastly - is this parameter required or optional?